lib/rails_admin/adapters/active_record/association.rb

```diff
@@ -39,6 +39,7 @@ def foreign_key
 
         def foreign_key_nullable?
           return if foreign_key.nil? || type != :has_many
+          return true if association.through_reflection
           klass.columns_hash[foreign_key.to_s].null
```

Comment:
This may return `nil` if key does not exist in `klass.columns_hash`. Perhaps not the return value we expect.
It should be better to write a test with a polymorphic association and see what happen (and fix potential issue), right ?
- [ ] Write test with polymorphic association